plugins/multilanguage.ts

```diff
@@ -176,7 +193,65 @@ export default function multilanguage(userOptions: Options) {
           document.head.appendChild(meta);
           document.head.appendChild(document.createTextNode("\n"));
         }
+
+        if (page.data.unmatchedLangUrl) {
+          appendHreflang("x-default", page.data.unmatchedLangUrl, document);
```

Comment:
This function should be used also to insert the other `<link>` elements.

Reply:
I would like to leave it (refactoring) for the next PR as 1 PR do 1 thing.